### DISCUSSION TITLE
Models below surface clipped at certain camera angles

### DISCUSSION THREAD

**Discussion by jonshutt** [RodrigoHamuy/react-three-map#98] [General]
Great library. I've been playing with adding models that go below the sea surface. 

When viewed at an angle - it renders the model - this is fine, we can apply a transparent 'surface' at sea level
When the move the camera and view it more top down, the model gets clipped.

I'm wondering if it's some depth/distance clipping in the mapbox camera?

https://github.com/RodrigoHamuy/react-three-map/assets/1441033/5743728f-5ce0-4138-b85c-3fcf324c31d6

**Comment by RodrigoHamuy** [comment]
Is a tricky one. At the moment the `Camera.far` attribute is calculated based on the projection matrix we receive from Mapbox/Libre, so not sure how to approach this problem. I could try decreasing `Camera.near` and see if that helps? 

Do you have a sandbox to play on? Thanks!

https://github.com/RodrigoHamuy/react-three-map/blob/main/src/core/sync-camera.ts#L30
